src/main/java/Avocado - Copy (31).java

```diff
@@ -0,0 +1,18 @@
+import java.util.Scanner;
+public class Avocado {
+    public static void main(String[] args) {
+        System.out.println("Hello from avocado");
+        System.out.println("How can I help you ?");
+        System.out.println("See you!");
+        while (true) {
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+            if (line.equals("bye")) {
+                return;
+            } else {
+                System.out.println(line);
+            }
+        }
+    }
+}
```

Comment:
I think you could afford to add some spacings in between lines of code that perform different functions. This makes the code more readable.
```suggestion
    public static void main(String[] args) {
    
        System.out.println("Hello from avocado");
        System.out.println("How can I help you ?");
        System.out.println("See you!");
        
        while (true) {
        String line;
        Scanner in = new Scanner(System.in);
        line = in.nextLine();
        
            if (line.equals("bye")) {
                return;
            } else {
                System.out.println(line);
            }
            
        }
    }
}
```